app/misc/git.ts

```diff
@@ -387,6 +391,14 @@ function pullFromRemote() {
 
 function pushToRemote() {
   let branch = document.getElementById("branch-name").innerText;
+
+  // Check to make sure we are not pushing without first committing files
+  if (pushButNoCommit == 0) {
+    console.log("There are no staged files")
```

Comment:
This error message is a bit misleading. Shouldn't it be "There are no commits available to push"?

Reply:
Good point @colinwerner ! I've updated the message to reflect your suggestion. My group will make sure to create a new issue after class.